frontend/spec/features/agents_spec.rb

```diff
@@ -0,0 +1,1728 @@
+        find('button', text: 'Save Person', match: :first).click
+        element = find('.alert.alert-success.with-hide-alert')
+        expect(element.text).to eq "Agent Created"
+
+        expect(find('#agent_names__0__use_dates__0__structured_date_single__date_expression_').value).to eq '1973'
+      end
+
+      it 'can add a parallel name to an Agent' do
+        now = Time.now.to_i
+
+        click_on 'Create'
+        click_on 'Agent'
+        click_on 'Person'
+
+        element = find('#agent_names__0__primary_name_')
+        element.fill_in with: "Agent Name #{now}"
+
+        click_on 'Add Parallel Name'
+        fill_in 'agent_names__0__parallel_names__0__primary_name_', with: "Primary Part of Name #{now}"
+
+        # Click on save
+        find('button', text: 'Save Person', match: :first).click
+        element = find('.alert.alert-success.with-hide-alert')
+        expect(element.text).to eq "Agent Created"
+
+        expect(find('#agent_names__0__parallel_names__0__primary_name_').value).to eq "Primary Part of Name #{now}"
+      end
+
+      it 'can add a name use date to a parallel name' do
+        now = Time.now.to_i
+
+        click_on 'Create'
+        click_on 'Agent'
+        click_on 'Person'
+
+        element = find('#agent_names__0__primary_name_')
+        element.fill_in with: "Agent Name #{now}"
+
+        click_on 'Add Parallel Name'
+        fill_in 'agent_names__0__parallel_names__0__primary_name_', with: "Primary Part of Name #{now}"
+
+        within '#agent_names__0__parallel_names_' do
+          click_on 'Add Use Date'
+        end
+
+        select 'Single', from: 'agent_names__0__parallel_names__0__use_dates__0__date_type_structured_'
+        fill_in 'agent_names__0__parallel_names__0__use_dates__0__structured_date_single__date_expression_', with: '1973'
+
+        # Click on save
+        find('button', text: 'Save Person', match: :first).click
+        element = find('.alert.alert-success.with-hide-alert')
+        expect(element.text).to eq "Agent Created"
+
+        expect(find('#agent_names__0__parallel_names__0__primary_name_').value).to eq "Primary Part of Name #{now}"
+        expect(find('#agent_names__0__parallel_names__0__use_dates__0__structured_date_single__date_expression_').value).to eq '1973'
+      end
+
+      it 'can add gender to an Agent' do
+        now = Time.now.to_i
+
+        click_on 'Create'
+        click_on 'Agent'
+        click_on 'Person'
+
+        element = find('#agent_names__0__primary_name_')
+        element.fill_in with: "Agent Name #{now}"
+
+        click_on 'Add Gender'
+        select 'Not Specified', from: 'agent_agent_genders__0__gender_'
+
+        # Click on save
+        find('button', text: 'Save Person', match: :first).click
+        element = find('.alert.alert-success.with-hide-alert')
+        expect(element.text).to eq "Agent Created"
+
+        expect(find('#agent_agent_genders__0__gender_').value).to eq 'not_specified'
+      end
+
```

Comment:
The `'can add date to a gender'` spec was not included here, see https://github.com/archivesspace/archivesspace/blob/v3.5.0/frontend/spec/selenium/spec/agents_spec.rb#L584